.github/workflows/playwright.yml

```
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-playwright:
+    timeout-minutes: 60
```

This timeout seems very high. If a test hangs, it will hang for an entire hour before failing. This delays feedback to human programmers and eats up our CI bandwidth. Given how small this app is (and thereby fast and responsive), I suggest a much smaller setting here. 

```suggestion
    timeout-minutes: 1
```

---

I think the timeout was this high because in ory/docs it runs a lot of examples and docker composes which take long sometimes. 
I will commit this but we can adjust down the line.

---

My bad, I mistook this as an internal timeout it Playwright. This is the timeout for the entire test. You are correct in setting it to something higher. The current value of 10 minutes makes sense.